python/tester/upload_from_repo.py

```diff
@@ -148,12 +149,13 @@ def main():
         for visit in visit_list:
             group = increment_group(instrument, group, 1)
             refs = prepare_one_visit(kafka_url, group, butler, instrument, visit)
+            uris = [butler.getURI(ref) for ref in refs]
```

Comment:
In theory `butler.get_many_uris` is faster than calling getURI in a loop (for direct butler it will do a single database query).

Reply:
I wanted to use something like that, but I didn't see that in (daily) pipelines.lsst.io.  Did something break the docs?

Reply:
https://pipelines.lsst.io/v/weekly/py-api/lsst.daf.butler.LimitedButler.html#lsst.daf.butler.LimitedButler.get_many_uris

Reply:
I see, it's in the base class.  A shame it doesn't get propagated to the derived.